app/views/child_benefit_tax/_results.html.erb

```diff
@@ -1,7 +1,7 @@
 <% results_html = capture do %>
   <div class="results-estimate">
-    <h3>Child Benefit received</h3>
-    <p class="numbers">
+    <h3 class="govuk-heading-m">Child Benefit received</h3>
```

Comment:
Could you use the heading component with `font_size: 19` here?

Reply:
Ah yes, good spot.